### ISSUE TITLE
Remove references (inc inside figs) to starting jobs on the bastion node

### ISSUE THREAD

**Issue by adamltyson** [neuroinformatics-unit/HowTo#66]
When logging into the bastion node it prints:
```
This is NOT an HPC node.
It   is NOT for running jobs.
```

So I assume we shouldn't promote the use of this node to start jobs.

**Comment by niksirbi** [comment]
What do you mean by "starting jobs"?
Submitting jobs (`sbatch`) is very much allowed, right?

**Comment by niksirbi** [comment]
I distinctly remember IT saying that it's ok to submit jobs from there, just not do anything else.

**Comment by adamltyson** [comment]
Me too, but the message is saying (or at least inferring) something else. Regardless of which is correct, I think it might be good to simplify our docs and present one single way of doing things.

**Comment by niksirbi** [comment]
As in, always go to `hpc-gw1` first?

**Comment by adamltyson** [comment]
Yeah, that's the way someone on the SWC network would do it anyway.
